apps/dashboard/src/main.tsx

```diff
@@ -277,13 +278,21 @@ const router = createBrowserRouter([
                     element: <ChannelPreferences />,
                     path: ROUTES.EDIT_WORKFLOW_PREFERENCES,
                   },
+                  {
+                    path: ROUTES.TRIGGER_WORKFLOW,
+                    element: (
+                      <ProtectedRoute permission={PermissionsEnum.EVENT_WRITE} isDrawerRoute>
+                        <TestWorkflowDrawerPage />
+                      </ProtectedRoute>
+                    ),
+                  },
                 ],
               },
               {
                 path: ROUTES.TEST_WORKFLOW,
                 element: (
                   <ProtectedRoute permission={PermissionsEnum.EVENT_WRITE}>
-                    <TestWorkflowPage />
+                    <TestWorkflowRouteHandler />
```

Comment:
Intermediate state for the feature flag, this page and layour will be removed in future iterations